app/script/view_model/content/HistoryExportViewModel.js

```diff
@@ -86,7 +86,7 @@ z.viewModel.content.HistoryExportViewModel = class HistoryExportViewModel {
       this.numberOfProcessedRecords(0);
       this.backupRepository
         .generateHistory(this.onProgress.bind(this))
-        .then(archive => archive.generateAsync({type: 'blob'}))
+        .then(archive => archive.generateAsync({compression: 'DEFLATE', type: 'blob'}))
```

Comment:
Which other compressions are there? Can we put them into a constant?

Reply:
Does this mean, that before this it wasn't compressed at all? The default value seems to be just 'STORE'.

Reply:
Only `STORE` (no compression) and `DEFLATE`
https://stuk.github.io/jszip/documentation/api_jszip/generate_async.html#compression-and-compressionoptions-options

Reply:
A constant would feel a bit overkill for the moment (since it's not used anywhere else). But it's open for debate :)

Reply:
@AndyLnd yes precisely :grimacing: @mythsunwind discovered this while comparing the backup he was generating VS. the backup generated by the app

Reply:
@atomrc Ok, I agree with you.